www/front_src/src/Resources/Listing/useLoadResources/index.ts

```diff
@@ -132,6 +134,18 @@ const useLoadResources = (): LoadResources => {
     };
   }, []);
 
+  React.useEffect(() => {
+    setAreDetailsEmpty(isNil(details));
+  }, [details]);
+
+  React.useEffect(() => {
+    if (areDetailsEmpty) {
+      return;
+    }
+
+    initAutorefresh();
+  }, [areDetailsEmpty]);
+
```

Comment:
Is it to dalay the refresh when opening the panel? why would we want to do that?

Reply:
I did that to fix the auto-reload of the details because it did not work until I click on the "refresh" button

Reply:
<img width="795" alt="Screenshot 2021-08-02 at 22 33 08" src="https://user-images.githubusercontent.com/12515407/127920274-41d3f8af-232a-4472-a756-5e963b3a511c.png">
Here are the network request when loading the job -> reloading the resource(s) passively